src/car.js

```diff
@@ -0,0 +1,34 @@
+import { CarError } from './errors/carError.js'
+
+export class Car {
+  static INITIAL_POSITION = 0
+  static MOVE_DISTANCE = 1
+  static CAR_NAME_MAX_LENGTH = 5
```

Comment:
```suggestion
  static NAME_MAX_LENGTH = 5
```
어차피 Car의 상수라서 CAR 라는 Prefix는 없어도 무방해보여요!